pilot/pkg/config/kube/gateway/testdata/invalid.status.yaml.golden

```diff
@@ -511,6 +511,7 @@ status:
       type: ResolvedRefs
     controllerName: istio.io/gateway-controller
     parentRef:
+      group: ""
```

Comment:
are we suppose to support `"core"` as well? I think I saw it in some examples - I think those were invalid, though?

Reply:
it is ""